### PR TITLE
adding and updating Readme

### PR DESCRIPTION
**Pull request by schwesig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,148 @@
...
+
+
+### Approved by Stakeholder
+tbd
+
+#### During the [Sprint]
+tbd
+
+### Redefine Issues & Epics
+Goal of [Redefine Issues & Epics] is to understand issues better if needed and split issues bigger than 13 Story Points (= Epics) into smaller issues (<= 13 Story Points).
```
Comment (goern) [inline README.md]: this is what we call 'backlog refinement session' in Thoth's T&C document? https://github.com/thoth-station/core/blob/master/docs/TermsAndConditionsForTheScrum.md#backlog-refinement-session

**Comment by durandom** [comment]
/hold

I want to start with a bare minimum as in the current README and increase the complexity only as needed and as questions arise

File: README.md [hunk 2]
```diff
@@ -1,46 +1,148 @@
...
 
-This repository contains information on how we implement SCRUM and holds issues for
-sprints. Sprints are planned in the [SCRUM BOARD](https://github.com/orgs/open-services-group/projects/21/views/1)
+This repository contains information on how we implement SCRUM and holds issues for sprints.
+Sprints are planned in the [SCRUM BOARD].
 
 # TL;DR;
 
-1. check the stakeholder backlog for assigned issues and prioritize completing them
+1. check the stakeholder backlog for issues assigned to you and well define them, that they can be prioritized
```
Comment (HumairAK) [inline README.md]:
```suggestion
1. check the stakeholder backlog for issues assigned to you and well define them so that they can be prioritized
```

File: README.md [hunk 3]
```diff
@@ -1,46 +1,148 @@
...
+The [Product Owner] reviews them regularly and decides, which of these need more attention, need more details and/ or need to be worked on (prioritized).
+
+  1. all issues start in the Stakeholder Backlog
+  1. all new and unassigned issues are reviewed by the [product owner] ([PO])
+  1. the product owner decides if an issue is well defined and needs to be prioritized
+     1. if an issue is well defined, the [PO] sets the label to `triage/accepted`
+     1. if an issue is not well defined, the [PO] assigns it to a contributor e.g. for decscribing it better or get more details
+        1. This loop continues until the [PO] sets the label `triage/accepted` and leaves it unassigned
+     1. if an issue is well defined `triage/accepted` and is prioritized by the [PO], the [PO] moves it to the [Product Backlog]
+  1. all assigned issues have to be completed by the assignee and unassigned once ready for review
```
Comment (HumairAK) [inline README.md:29]: > all assigned issues have to be completed by the assignee and unassigned once ready for review

a bit unclear to me, are you referring to when the PO assigns it to a contributor in  point 3-ii ?

File: README.md [hunk 4]
```diff
@@ -1,46 +1,148 @@
...
+The goal of the [Product Backlog] is to have a collection of well defined (`triage/accepted`) and ready to be worked on issues.
+ - Issues in the [Product Backlog] are prioritized and need to be worked on in the next sprints.
+ - During the [Sprint Planning] these issues can be moved to the [Sprint Backlog].
+ - The [Product Backlog] holds only issues;
+these issues must be `triage/accepted`, but they don't need to be "assigned" to any person yet.
+
+### In Progress
+The goal of [In Progress] is to see, which issues are currently worked on.
+ - During the [Sprint Planning] issues are assigned and moved to the [Sprint Backlog].
+ - All issues in the [Sprint Backlog] while be worked on in during the [Sprint].
```
Comment (HumairAK) [inline README.md]:
```suggestion
 - All issues in the [Sprint Backlog] will be worked on during the [Sprint].
```

File: README.md [hunk 5]
```diff
@@ -1,46 +1,148 @@
...
+ - When an assignee actively works on one or more of his/her/* issues, the assignee moves the issue to [In Progress].
+
+### Done
+The goal of [Done] is to collect all the solved issues.
+ - Issues here are ready to be tested/ checked/ approved by the Stakeholder (creator of the issue in general). 
+ - Solved is defined as: the assignee took care of all the mentioned acceptance criterias during the sprint.
```
Comment (HumairAK) [inline README.md]:
```suggestion
 - Solved is defined as: the assignee took care of all the mentioned acceptance criteria during the sprint.
```

File: README.md [hunk 6]
```diff
@@ -1,46 +1,148 @@
...
+ - The assignee defines solved and moves the issue from [In Progress] to [Done].
```
Comment (HumairAK) [inline README.md:48]: >   - The assignee defines solved and moves the issue from [In Progress] to [Done].

not sure what you mean by "assignee defines solved"

File: README.md [hunk 7]
```diff
@@ -1,46 +1,148 @@
...
+## Glossary
+PO - Product Owner
+well defined issue - an issue that has enough details, understanding of the problem, that it can be used in the [Product Backlog](#product-backlog)
+Story Points - Effort estimation for an issue.  [Glossary] 
+
+## The Scrum Board
+[SCRUM BOARD]
+| Stakeholder Backlog | Product Backlog | Sprint Backlog | In Progress | Done | Approved by Stakeholder | Closed |
+|---------------------|-----------------|----------------|-------------|------|-------------------------|--------|
+|                     |                 |                |             |      |                         |        |
```
Comment (HumairAK) [inline README.md]: not sure I see the point of this markdown table  :thinking:

File: README.md [hunk 8]
```diff
@@ -1,46 +1,148 @@
...
 
+#### Sprint Backlog
+
 ## Product Owner
+The [Product Owner] is repsonsible for:
+ - Priorizing issues: Move it from [Stakeholder Backlog] to [Product Backlog]
+ - Well defined: Approve an issue or Request a better defintion --> set `triage/accepted` label
+ - Check or delegate the [Approval by Stakeholder]
 
 @goern and @durandom are product owners
```
Comment (HumairAK) [inline README.md]: I don't think @mentions work in the markdown, maybe provide a link?

File: README.md [hunk 9]
```diff
@@ -1,46 +1,148 @@
...
 1. issues in the `Done` column with a -1 are discussed and all other issues are closed out
 2. issues in the `Product Backlog` are assigned and moved to the `Sprint Backlog`
 
+#### Sprint Backlog
+
 ## Product Owner
+The [Product Owner] is repsonsible for:
+ - Priorizing issues: Move it from [Stakeholder Backlog] to [Product Backlog]
+ - Well defined: Approve an issue or Request a better defintion --> set `triage/accepted` label
+ - Check or delegate the [Approval by Stakeholder]
```
Comment (HumairAK) [inline README.md]:
```suggestion
 - Check or delegate the [Approved by Stakeholder]
```

File: README.md (hunk 3 above)
Comment (schwesig) [reply README.md:29]: yes.
Maybe this needs to be removed (redundant), or updated.
I am not sure, what is the best here yet.
Thanks for bringing it up.

File: README.md (hunk 1 above)
Comment (schwesig) [reply README.md]: agreed, will change it to this name

File: README.md [hunk 10]
```diff
@@ -1,46 +1,148 @@
...
+These are just general guidelines. Over time each team can find its own rules and time estimates ([Velocity])
+
+
+### Approved by Stakeholder
+tbd
+
+#### During the [Sprint]
+tbd
+
+### Redefine Issues & Epics
```
Comment (schwesig) [inline README.md]:
```suggestion
### Backlog Refinement Session
```

Comment (schwesig) [reply README.md]: changed @goern

File: README.md (hunk 9 above)
Comment (schwesig) [reply README.md]: Thanks, add change

File: README.md (hunk 8 above)
Comment (schwesig) [reply README.md]:
```suggestion
[@goern](https://github.com/goern) and [@durandom](https://github.com/durandom) are product owners
```

File: README.md (hunk 7 above)
Comment (schwesig) [reply README.md]: 1. to see the board columns without opening the board (I thought it might be easier seeing it on the same page)
2. maybe for adding basic rules in the colums data area, like "must be `triage accepted` or must be assigned ...

File: README.md (hunk 6 above)
Comment (schwesig) [reply README.md:48]: Maybe a lack of a better wording.
What I like to say:
An issue is a problem that should be solved, or creates an MVP, or has acceptance criteria.
An assignee works on these things and "solves" them, or creates results, that fit the criteria.
When this happened, and the assignee decides this, the issue can move on column further, to the Done column.
Bascically: done = solved (from the assigne's view)
If the issue is done/ solved in the stakeholders view, will be the next step.

File: README.md (hunk 2 above)
Comment (schwesig) [reply README.md]: thanks, added change

File: README.md [hunk 11]
```diff
@@ -1,46 +1,148 @@
...
+[Sprint Planning / Review meeting]: #sprint-planning-/-review-meeting
+[Sprint Planning]: #sprint-planning
+[Sprint]: #sprint
+[In Progress]: #in-progress
+[Done]: #done
+[Approved by Stakeholder]: #approved-by-stakeholder
+[The Scrum Board]: #the-scrum-board
+[Story Points Estimates]: https://github.com/thoth-station/core/blob/master/docs/TermsAndConditionsForTheScrum.md#estimates
+[Velocity]: https://www.scrum.org/resources/blog/agile-metrics-velocity
+[Redefine Issues & Epics]: #redefine-issues-&-epics
```
Comment (schwesig) [inline README.md]:
```suggestion
[Backlog Refinement Session]: #backlog-refinement-session
```

File: README.md (hunk 1 above)
Comment (schwesig) [inline README.md]:
```suggestion
Goal of [Backlog Refinement Session] is to understand issues better if needed and split issues bigger than 13 Story Points (= Epics) into smaller issues (<= 13 Story Points).
```

**Comment by schwesig** [comment]
I went trough all comments and suggestions, and accpeted most of them. and batch commited them... but it didn't work.
User failure... I will try again

**Comment by schwesig** [comment]
worked now, changes applied/accpeted
thanks for the feedback @HumairAK @goern

File: README.md [hunk 12]
```diff
@@ -1,46 +1,148 @@
...
+well defined issue - an issue that has enough details, understanding of the problem, that it can be used in the [Product Backlog](#product-backlog)
+Story Points - Effort estimation for an issue.  [Glossary] 
+
+## The Scrum Board
+[SCRUM BOARD]
+| Stakeholder Backlog | Product Backlog | Sprint Backlog | In Progress | Done | Approved by Stakeholder | Closed |
+|---------------------|-----------------|----------------|-------------|------|-------------------------|--------|
+|                     |                 |                |             |      |                         |        |
+
+
```
Comment (schwesig) [inline README.md]:
```suggestion
# Further reading
The Thoth team practiced SCRUM and maintains their [Terms and Conditions](https://github.com/thoth-station/core/blob/master/docs/TermsAndConditionsForTheScrum.md), which we can build upon.
```

File: README.md (hunk 6 above)
Comment (schwesig) [reply README.md:48]: I guess and hopem @HumairAK 's `thumbs up`is an OK, and I will call this resolved.

**Comment by schwesig** [comment]
/close
cleanup after change of organisation
